docs/getting-started-guides/libvirt-coreos.md

```diff
@@ -117,6 +117,26 @@ setfacl -m g:kvm:--x ~
 
 By default, the libvirt-coreos setup will create a single Kubernetes master and 3 Kubernetes nodes. Because the VM drives use Copy-on-Write and because of memory ballooning and KSM, there is a lot of resource over-allocation.
 
+There are two major ways of how-to run and configure the libvirt-based Kubernetes cluster on CoreOS instances:
```

Comment:
There is both an automated way and a manual, customizable way of setting up libvert Kubernetes clusters on CoreOS.